test_parse.py

```diff
@@ -8,6 +8,30 @@
 from ast_comments import Comment, parse
 
 
+def test_all_comment_places_in_tree():
```

Comment:
Can you refactor the test, please? Current test does not test all-comments-in-place. It tests only that there are 7 nodes at the module level. 
When testing "Comment 4" position there's no need in Comment 1,2,7,8, and related prints.

I would split it to smaller tests covering specific details of your commit
Additionally, can you add test showing that if "Comment 4" has indent lower than "print('2')", then it goes to "else" block? This behavior is not a straightforward one so it better be documented somehow.

Reply:
added tests for the several placements of comments which were not covered and removed the big test.

also added tests to show behaviour on wrong indentation of comments within tests